docs/project/changelog.md

```diff
@@ -18,6 +18,8 @@ myst:
 
 - ABI break: Upgraded Emscripten to 3.1.63 {pr}`5343` {pr}`5350` {pr}`5357`
   {pr}`5334` {pr}`5363`
+- ABI break: Switched to using WebAssembly exception handling for C++ errors,
+  Rust panics, and setjmp/longjmp. {pr}`5320`
```

Comment:
Here's my attempt at documenting it better:

```suggestion
- ABI break: Switched to using WebAssembly exception handling for C++ errors,
  Rust panics, and setjmp/longjmp. Projects and build system helpers that have previously set the
`-fexceptions` compilation flag **must** switch to using `-fwasm-exceptions`. {pr}`5320`
```

Reply:
Yeah we can intercept most things but not all things in pywasmcross.

Is it a breaking change for cibuildwheel? Does it automatically pick up new Pyodide versions and start building for them or is some intervention needed? In any case we can just fix it.

Reply:
It's not a breaking change for `cibuildwheel`, but it'll be one for `pybind11` as it tests for WASM builds downstream (and Cython as well, should they choose to test WASM later on).

`cibuildwheel` doesn't actually pick up new Pyodide versions as they are released at the moment, yes. It's usually me who keeps updating it there. My PR for allowing arbitrary Pyodide versions is still open; I think I should revisit it.

Reply:
It's pretty easy to hack on. You can probably just push your changes to my PR:
https://github.com/pyodide/pyodide-build/pull/81/files

Reply:
Or maybe we should leave it to a follow-up pr.